_rules/SC1-1-1-accessible-name-is-image-file-name.md

```diff
@@ -0,0 +1,64 @@
+---
+name: accessible name is image file name
+description: |
+  This rule checks that the accessible name of an image is not its file name.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML `img` element that is exposed to assistive technologies, where the element has a non-empty `alt` attribute that is equal to the accessible name of the element, and where the `alt` attribute of the element references the file name and extension of the `src` attribute of the element.
```

Comment:
"references the file name and extension" is not unambiguous for me. 

I do not understand what makes test case 1 applicable and test case 3 inapplicable. They both "reference" the file name (example) and the extension (.pdf and PDF). Is "reference" only applicable if both file name and extension are used in the same case as the one used in the src attribute? Or does it have to include the dot in the extension?

Reply:
That's a really good point and something we'll have to clarify.